tests/pipeline/test_examples.py

```diff
@@ -1,10 +1,14 @@
 import importlib
+import os
+import sys
 
 import pytest
 
 import tests.utils as utils
 
-dot_path_to_addon = utils.get_dot_path_from_tests_to_current_dir(__file__)
+sys.path.append(os.path.abspath(f"examples/{utils.get_path_from_tests_to_current_dir(__file__)}"))
```

Comment:
This line can be remove

Reply:
It's can't be. Please, see the latest resolved issue in PR #21.